site/pages/docs/ref/multimedia/multimedia-fr.hbs

```diff
@@ -115,15 +115,15 @@
 
 			<section>
 				<h5>Code</h5>
-				<pre><code>&lt;figure class="wb-mltmd" data-wb-mltmd='{"shareUrl": "http://youtu.be/p2-1ZIHu1W0"}'&gt;
-	&lt;video poster="http://www.servicecanada.gc.ca/fra/video/images/te-lj-fra.jpg" title="Trouver un emploi"&gt;
+				<pre><code>&lt;figure class="wb-mltmd" data-wb-mltmd='{"shareUrl": "https://wet-boew.github.io/v4.0-ci/demos/multimedia/multimedia-fr.html"}'&gt;
```

Comment:
Does a relative link work for this?

Reply:
Unfortunately not. The share widget seems to use ``shareUrl``'s value "as-is".